### PR TITLE
Add network stats

### PR DESCRIPTION
**Pull request by ZeroMagic**
Ref: #80 
Actually, containerd has no concept of networks. Then containerd only needs to transfer the network stats to CRI or other receivers. So just add the netstat stuff to the cgroups repo for now.

Signed-off-by: ZeroMagic <anthonyliu@zju.edu.cn>

### CODE REVIEW

File: metrics.proto [hunk 1]
```diff
@@ -121,3 +122,15 @@ message RdmaEntry {
 	uint32 hca_handles = 2;
 	uint32 hca_objects = 3;
 }
+
+message NetworkStat {
+	string name = 1;
+	uint64 RxBytes = 2;
```
Comment (crosbymichael) [inline metrics.proto]: these should be lowercase:

`rx_bytes` `tx_bytes`

Comment (ZeroMagic) [reply metrics.proto]: done.

**Comment by crosbymichael** [comment]
LGTM

**Comment by ZeroMagic** [comment]
@fuweid Can you help with the second round of review?

**Comment by fuweid** [comment]
maybe we can put the proto version in READMD.md so that it is easy to make the right pb.go file. :p

**Comment by ZeroMagic** [comment]
The version of gogo/protobuf I use is v0.5. Its commit id is 342cbe0a04158f6dcb03ca0079991a51a4248c02.

**Comment by stormgbs** [comment]
LGTM

**Comment by fuweid** [comment]
The pb file is correct and it is the same with https://github.com/containerd/containerd/blob/master/script/setup/install-protobuf. 

ping @AkihiroSuda could you help us to review it again? thanks

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**